src/SDKs/Storage/Management.Storage/Generated/Models/Action.cs

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for
+// license information.
+//
+// Code generated by Microsoft (R) AutoRest Code Generator 1.0.0.0
+// Changes may cause incorrect behavior and will be lost if the code is
+// regenerated.
+
+namespace Microsoft.Azure.Management.Storage.Models
+{
+    using Azure;
+    using Management;
+    using Storage;
+    using Newtonsoft.Json;
+    using Newtonsoft.Json.Converters;
+    using System.Runtime;
+    using System.Runtime.Serialization;
+
+    /// <summary>
+    /// Defines values for Action.
+    /// </summary>
+    [JsonConverter(typeof(Newtonsoft.Json.Converters.StringEnumConverter))]
+    public enum Action
```

Comment:
"Deny" could be a possible value to be added for disable . That's reason to set it as a enum here. 
I already set default value as "Allow" in swagger, https://github.com/Azure/azure-rest-api-specs/blob/master/arm-storage/2017-06-01/swagger/storage.json#L885. But with "x-ms-enum" extension, it seems default value is not set. And "x-ms-enum" is required for swagger's linter check.